scripts/keymap.lua

```diff
@@ -5,12 +5,12 @@ kmp = {}
 -- Microbe Editor --
 
 kmp.undo = {"ctrl", "Z"}
-kmp.redo = {"ctrl", "R"}
+kmp.redo = {"ctrl", "Y"}
```

Comment:
I switched it because people who I have seen play on youtube so far were assuming the windows configuration and being confused when ctrl+z for undo etc doesnt work